File: 1550-three-consecutive-odds/1550-three-consecutive-odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        
        count = 0
        
        for x in arr:
            if count == 3:
                return True
            
            if x%2:
                count +=1
            else:
                count = 0
        
        return count==3
            

File: 1550-three-consecutive-odds/1550-three-consecutive-odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        int count = 0;  // Initialize count to keep track of consecutive odd numbers

        for (int i = 0; i < arr.length; i++) {
            if (arr[i] % 2 != 0) {  
                count++;  
                if (count == 3) {  
                    return true;
                }
            } else {  
                count = 0;
            }
        }
        return false;  
    }    
}